EscelForce.Business.Login/Services/MapExtraction/CreateExtractionMapService.cs

```diff
@@ -65,18 +66,22 @@ public ServiceResponseModel<bool> SubmitOnObjectSelection(string objectName)
                     Objects = new List<SfObject>()
                 };
 
-                if (!query.Objects.Any(x => x.Name == objectName))
+                string selectedObjectName = objectName.Contains('|')? objectName.Split('|')[1].Trim(): objectName.Trim();
+                string label = objectName.Contains('|') ? objectName.Split('|')[0].Trim() : objectName.Trim();
+
+                if (!query.Objects.Any(x => x.Name == selectedObjectName))
```

Comment:
Please use `string.Equals` for string comparison.

Reply:
Changed it